src/components/SettingsButton/SettingsButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { MALE_GENDER, FEMALE_GENDER } from '../../constants/gender';
+import { fontWeightBold, spacingSmall } from '../../styles/designTokens';
+import MaleButton from './MaleButton';
+import FemaleButton from './FemaleButton';
+
+const Label = styled.label`
+  font-weight: ${fontWeightBold};
+`;
+
+const ButtonGroup = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: ${spacingSmall} 0;
+`;
+
+function SettingsButton({ gender, onChange }) {
+  return (
+    <div>
+      <Label>Gender</Label>
+      <ButtonGroup>
+        <MaleButton checked={gender === MALE_GENDER} onChange={onChange} />
```

Comment:
Looks like `MaleButton` component is almost the same as `FemaleButton`.  The only difference is name and probably some styling. Components share the same context being toggle buttons. That's said, those components will _almost_ always be tied together (same look, same behaviour), rather than shared in different places.  I would go with single component that conditionally displays male or female and receive extended props from parent component like `id`, `value` or `value` . But that's just an idea 🙂